api/CanvasPattern.json

```diff
@@ -70,10 +70,10 @@
               "version_added": false
             },
             "opera": {
-              "version_added": "9"
+              "version_added": "55"
```

Comment:
Are you sure about this? https://github.com/mdn/browser-compat-data/blob/f3d419658370750830c5e8450922cba17b311000/api/CanvasRenderingContext2D.json#L1046-L1051 says `CanvasPattern` was supported in Presto. Maybe we need to test an instance of `CanvasPattern` for this?

Reply:
Good point; I'll recreate this PR using the collector instead, adding a custom test!